authorization.go

```diff
@@ -58,6 +60,15 @@ func authorize(r *http.Request, jwtConfig *jwtConfig, publishOrigins []string, c
 		return validateJWT(authorizationHeaders[0][7:], jwtConfig)
 	}
 
+	authorizationQuery, queryExists := r.URL.Query()["authorization"]
```

Comment:
You could probably merge with this line with the next one.

Reply:
done.